modules/pre-commit.nix

```diff
@@ -366,7 +366,7 @@ in
           if ! type -t git >/dev/null; then
             # This happens in pure shells, including lorri
             echo 1>&2 "WARNING: pre-commit-hooks.nix: git command not found; skipping installation."
-          elif [[ ! -e .git ]]; then
+          elif ! ${pkgs.locate-dominating-file}/bin/locate-dominating-file .git >/dev/null; then
```

Comment:
This should probably be written with `git rev-parse --git-dir` instead of using `locate-dominating-file` which may fail or become slow at deep folder structures due to the expectation to find a folder named `.git`.

Reply:
I was trying to understand the nix runtime dependencies of this generated script, but I'm unable to find any via `nix-store --query`. 

We must have the assumption that `git` is installed already, so I don't think avoiding depending on it gains us anything in this context.

All this to say, I agree @imsofi, let me change that to the `git` command.